test/Core.Test/Billing/Organizations/Commands/UpdateOrganizationLicenseCommandTests.cs

```diff
@@ -99,6 +100,139 @@ await sutProvider.GetDependency<IOrganizationService>()
         }
     }
 
+    [Theory, BitAutoData]
```

Comment:
maybe we should add more test  like :

- Null ClaimsPrincipal
- Wrong installation ID in claims
- Expired license with valid claims